Szwalnia/Szczegoly.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace Szwalnia
+{
+    public partial class Szczegoly : Form
+    {
+        public Szczegoly(SzwalniaEntities db, int sourcesidPP)
+        {
+            InitializeComponent();
+            lblID.Text = sourcesidPP.ToString();
```

Comment:
na pewno wyświetlenie ID to dobry pomysł? nie mamy bardziej czytelnej informacji dla użytkownika?